### ISSUE TITLE
The character runs very fast, and pressing the "run" (shift) does not change the speed

### ISSUE THREAD

**Issue by WNC12k** [Sumwunn/DoomRPG#50]
The character runs very fast, and pressing the "run" (shift) does not change the speed. This bug has existed for a long time. What do you think about it? From-and what it can be?

**Comment by FoxMcloud5655** [comment]
Not exactly sure what you're reporting...  Is your auto-run setting set to always run?

**Comment by WNC12k** [comment]
autorun is disabled. However, the character always runs. You don't have this bug?

**Comment by devilemporium** [comment]
this isn't a bug (i'm pretty sure) but the character always runs due to agility being a stat in drpg

**Comment by Sumwunn** [comment]
I've noticed this same issue with Brutal Doom Black Edition. 
Perhaps a rework in that department is required.

**Comment by FoxMcloud5655** [comment]
The way I have my config setup is that I run unless shift is held.  Then I walk and auto-sell anything I pick up.

**Comment by Demiosis** [comment]
You can set max char move speed in the options as well as max weapon speed! (Agi stats won't go over this threshold! )

**Comment by WNC12k** [comment]
Yes, thank you. I had to configure the following bind via config:

Shift=drpg_maxspeed 0.8
Alt=drpg_maxspeed 0.45

But these are crutches. I would like it to be inside the gameplay

**Comment by Sumwunn** [comment]
At least with a joystick, if strafing causes double speed, it can be heavily mitigated by setting "Moving Forward" sensitivity to 0.6 (lower breaks joystick detection, I think).
